### ISSUE TITLE
Add support for coming spread syntax

### ISSUE THREAD

**Issue by cvle** [palantir/tslint#1715]
### Bug Report

- __TSLint version__: `4.0.0-dev.1`
- __TypeScript version__: `2.2.0-dev.20161113`
- __Running TSLint via__: vim

#### TypeScript code being linted

```ts
const Component: React.StatelessComponent<Props> = ({ ...rest }) => <div {...rest } />;
```

#### Actual behavior

`Missing semicolon (semicolon) [typescript/tslint]`

#### Expected behavior

Pass

**Comment by cvle** [comment]
Oops I figured that my vim used a different tslint version. Sorry for the noise. Closing this for now.
